lib/tasks/address.rake

```diff
@@ -0,0 +1,91 @@
+# encoding: utf-8
+
+#  Copyright (c) 2012-2020, CVP Schweiz. This file is part of
+#  hitobito_cvp and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_cvp.
+
+namespace :address do
+  desc 'Import Post Addresses'
+  task :import => [:environment] do
+    Address::Importer.new.run
+  end
+
+  class Overview
+    def initialize(file)
+      @file = file
+    end
+
+    def addresses
+      @addresses ||= Rails.root.join('tmp/addresses.csv')
+    end
+
+    def write_addresses
+      pipeline = [
+        "csvgrep -e latin1 -c Loeschflag -m ' ' -d '|' #{@file}",
+        "csvgrep -c Land -m 'CH'",
+        "csvgrep -c Strasse -m ' ' -i",
+        "csvgrep -c Postleitzahl -m ' ' -i",
+        "csvcut -c Strasse,Hausnummer,HausnummerZusatz,Postleitzahl,Land"
```

Comment:
Fehlt hier noch ein Hinweis, dass man `csvkit` installieren muss, um dieses Feature zu nutzen? Vermutlich müsste das im rake-task `address:check` passieren.

Reply:
Ich verschiebe das in den cvp wagon, da der check auf Basis ihrere CVS Files implementiert ist. Den Hinweis auf CVSKit werde ich ergänzen und die Overview Klasse in den Namespace `Address` verschieben aber im Rake file belassen, da sie nur für den spezifischen Task relevant ist